src/plugins/data/public/ui/ui_service.ts

```diff
@@ -38,30 +34,9 @@ export class UiService implements Plugin<IUiSetup, IUiStart> {
     this.enhancementsConfig = enhancements;
   }
 
-  public setup(core: CoreSetup, {}: UiServiceSetupDependencies): IUiSetup {
-    return {
-      __enhance: (enhancements?: UiEnhancements) => {
-        if (!enhancements) return;
-        if (enhancements.query && enhancements.query.language) {
-          this.queryEnhancements.set(enhancements.query.language, enhancements.query);
-        }
-        if (enhancements.queryEditorExtension) {
-          this.queryEditorExtensionMap[enhancements.queryEditorExtension.id] =
-            enhancements.queryEditorExtension;
-        }
-      },
-    };
-  }
+  public setup(core: CoreSetup, {}: UiServiceSetupDependencies): any {}
```

Comment:
this should be `IUiSetup` just need to update the type IUiSetup to be what you expect now in this case empty